lib/utils.js

```diff
@@ -29,6 +29,8 @@ const CODE_OWNERS_FILE_URL =
 const CHANGELOG_START = 'PR CHANGELOG';
 const SUCCESS_STATUS = 204;
 const THREE_MINUTES = 60 * 1000 * 3;
+let MIN_BUILD_DATE = new Date();
+MIN_BUILD_DATE.setDate(MIN_BUILD_DATE.getDate() - 2); // 2 days prior.
```

Comment:
I would just set the two days here, and count the exact date in `checkAndTagPRsWithOldBuilds`.

Reply:
I don't think I understand what you mean here.
Also, I'm setting it here so I will be able to mock it while testing.

Reply:
Oh, okay, fine then.